apps/condo/domains/resident/tasks/helpers/sendBillingReceiptNotifications.spec.js

```diff
@@ -3,18 +3,24 @@
  */
 
 const index = require('@app/condo/index')
+const dayjs = require('dayjs')
 
+const { getRedisClient } = require('@open-condo/keystone/redis')
 const { setFakeClientMode, setAllFeatureFlags } = require('@open-condo/keystone/test.utils')
 
 const { sendBillingReceiptNotifications } = require('./sendBillingReceiptNotifications')
 
 
+const REDIS_LAST_DATE_KEY = 'LAST_SEND_BILLING_RECEIPT_NOTIFICATION_CREATED_AT'
```

Comment:
This is a second definition of this constant in the code& What about importing it from a single place?